core/src/repair/cluster_slot_state_verifier.rs

```diff
@@ -902,7 +902,7 @@ pub(crate) fn check_slot_agrees_with_cluster(
             }
         }
 
-        datapoint_info!(
+        datapoint_error!(
```

Comment:
same with this, i think we should keep as info, `duplicate_slot` being `datapoint_error!` is probably good though.